File: nest-backend/src/courses/image/course-image.entity.ts
import {Entity, Column, PrimaryGeneratedColumn, ManyToOne } from "typeorm";
import {Course} from "../courses.entity";

@Entity()
export class CourseImage {

  @PrimaryGeneratedColumn()
  id:number;


  @Column("mediumtext")
  image: string;


  @ManyToOne(type => Course)
  course: Course;

  @Column()
  courseId:number;


}

File: jschool-spring/src/main/java/com/shak/jschool/entities/course/CourseImageEntity.java
package com.shak.jschool.entities.course;

import javax.persistence.*;


@Entity(name = "course_image")
public class CourseImageEntity {


    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;


    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "courses_image")
    private CourseEntity course;

    @Column(name="image",columnDefinition = "TEXT")
    private String image;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public CourseEntity getCourse() {
        return course;
    }

    public void setCourse(CourseEntity course) {
        this.course = course;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }
}
